league/manager/models.py

```diff
@@ -25,64 +23,83 @@ def played_matches(self):
         return self.match_set.filter(result__isnull=False).count()
 
     @property
-    def ended(self):
+    def is_ended(self):
         if self.all_matches > 0:
             return self.all_matches == self.played_matches
         else:
             return False
 
     @property
     def league_winner(self):
-        #TODO: zrobic winnera ligi ASAP
+        # TODO: Implement condition for winning a league ASAP
         pass
 
 
 class Team(models.Model):
+    league = models.ForeignKey(League, on_delete=models.CASCADE)
     team_name = models.CharField(max_length=10)
     matches_won = models.IntegerField(default=0)
     matches_draw = models.IntegerField(default=0)
     matches_lost = models.IntegerField(default=0)
-    league = models.ForeignKey(League, on_delete=models.CASCADE)
 
     def save(self, *args, **kwargs):
         if self.league.teams_number >= self.league.max_number_of_teams:
             raise ValidationError("Max number of teams exceeded", code="max_teams")
         return super().save(*args, **kwargs)
 
-    @property # TODO:Zuzannka77 add test to check if this property works properly
+    @property  # TODO:Zuzannka77 add test to check if this property works properly
     def sum_of_points(self):
-        return self.matches_won * self.league.points_for_win +\
-               self.matches_draw * self.league.points_for_draw +\
+        return self.matches_won * self.league.points_for_win + \
+               self.matches_draw * self.league.points_for_draw + \
                self.matches_lost * self.league.points_for_lost
 
-    @property # TODO:Zuzannka77 add test to check if this property works properly
+    @property  # TODO:Zuzannka77 add test to check if this property works properly
     def matches_team_played(self):
-        return self.matches_won +\
-               self.matches_draw +\
+        return self.matches_won + \
+               self.matches_draw + \
                self.matches_lost
 
 
 class Match(models.Model):
     league = models.ForeignKey(League, on_delete=models.CASCADE)
-    # date = models.DateTimeField(default=datetime.now, blank=True)
+    # match_date = models.DateTimeField(auto_now_add=True)
```

Comment:
Wow - second change in one PR that I adore - replaced possible confusing attribute name with more explicit one.